services/web/server/tests/integration/conftest.py

```diff
@@ -130,6 +130,9 @@ def _default_app_config_for_integration_tests(
     test_environ["WEBSERVER_LOGLEVEL"] = "WARNING"
     test_environ["OSPARC_SIMCORE_REPO_ROOTDIR"] = f"{osparc_simcore_root_dir}"
 
+    # NOTE: removed from webserver_environ after split with GC
```

Comment:
this note is not too clear for me. did you not split the GC already?

Reply:
yes, not very clear.

@GitHK The basic idea is that since we have this new plugin functionality, we will need to split the integration tests in the future and address every different "configured service" separately

SEE details in #2896